docs/config.md

```diff
@@ -860,13 +860,19 @@ notifications = true
 # You can optionally filter to specific notification types.
 # Available types are "agent-turn-complete" and "approval-requested".
 notifications = [ "agent-turn-complete", "approval-requested" ]
+
+# Disable the "approaching rate limits" model-switch prompt.
+# Warnings still appear in chat history. Defaults to true (prompt enabled).
+rate_limit_model_suggestions = false
 ```
 
 > [!NOTE]
 > Codex emits desktop notifications using terminal escape codes. Not all terminals support these (notably, macOS Terminal.app and VS Code's terminal do not support custom notifications. iTerm2, Ghostty and WezTerm do support these notifications).
 
 > [!NOTE] > `tui.notifications` is built‑in and limited to the TUI session. For programmatic or cross‑environment notifications—or to integrate with OS‑specific notifiers—use the top‑level `notify` option to run an external program that receives event JSON. The two settings are independent and can be used together.
 
+Set `rate_limit_model_suggestions` to `false` if you prefer to stay on your current model without Codex opening the "Switch to gpt-5-codex-mini" popup. `/status` and the inline warning history cells still reflect rate-limit data; the flag only suppresses the selection prompt.
```

Comment:
question: Should we also scope this to include rate limit warnings? I deliberately chose not to here, but I'm open to discussion.